src/components/fundamentals/input-header.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react"
+import InfoTooltip from "../info-tooltip"
+
+type InputHeaderProps = {
+  label: string
+  required?: boolean
+  withTooltip?: boolean
+  tooltipText?: string
+  tooltipProps?: any
+}
+
+const InputHeader: React.FC<InputHeaderProps> = ({
+  label,
+  required = false,
+  withTooltip = false,
+  tooltipText,
+  tooltipProps,
+}) => {
+  return (
+    <div className="w-full flex pb-1.5 font-semibold text-small leading-xsmall text-grey-50 items-baseline">
+      {label}
+      {required && <div className="text-red-600 "> *</div>}
```

Comment:
In the figma the color rose-50 is used for warnings/reds so I think we should also use this here 😁